index.html

```diff
@@ -17,24 +17,36 @@
                 <a href="/" class="logo-container">
                     <h3 class="logo-txt">Edwin G.</h3>
                 </a>
-                <nav class="menu-btn">
+                <nav class="menu-bar">
                     <img src="./imgs/Union.svg" alt="menu icon">
+                    <ul class="nav-menu">
+                        <li><a href="#portfolio">Portfolio</a></li>
+                        <li><a href="#about">About</a></li>
+                        <li><a href="#contact">Contact</a></li>
+                        <li><a href="#">
+                                <i class="fa fa-envelope"></i>
+                            </a></li>
+                    </ul>
                 </nav>
             </section>
         </header>
         <section class="headline">
-            <h1 class="primary">
-                Hey There. I'm Edwin
-            </h1>
-            <h2 class="supporting">
-                I’m a Software Developer
-            </h2>
-            <p class="description">
-                I can help you build a product , feature or website Look through some of my work and experience! If you
-                like what you
-                see and have a project you need coded,
-                don’t hesitate to contact me.
-            </p>
+            <span class="headline-text">
```

Comment:
As the previous reviewer requested, please fix the overlaying issue on the heading section 👍
<img width="795" alt="Screenshot 2023-01-28 at 11 15 01 PM" src="https://user-images.githubusercontent.com/25887253/215288968-cf13157c-3cfa-454a-b07a-20947edabef7.png">

Reply:
I have fixed the required changes. I will work on the Optional changes after the submission due to the deadline approaching in a few minutes. Making it fully responsive will take me some more time and I don't want to miss the deadline and repeat the week. Thanks.